ui/apps/platform/src/Containers/SystemHealth/UsageStatistics/UsageStatisticsForm.tsx

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+import React, { ReactElement, useEffect, useState } from 'react';
+import {
+    Alert,
+    Button,
+    DatePicker,
+    DescriptionList,
+    DescriptionListDescription,
+    DescriptionListGroup,
+    DescriptionListTerm,
+    Divider,
+    Grid,
+    GridItem,
+    yyyyMMddFormat,
+} from '@patternfly/react-core';
+import { MaxSecuredUnitsUsageResponse, SecuredUnitsUsage } from '../../../types/productUsage.proto';
+import {
+    downloadProductUsageCsv,
+    fetchCurrentProductUsage,
+    fetchMaxCurrentUsage,
+} from '../../../services/ProductUsageService';
+import { getAxiosErrorMessage } from '../../../utils/responseErrorUtils';
+
+function UsageStatisticsForm(): ReactElement {
+    const initialStartDate = new Date();
+    initialStartDate.setDate(initialStartDate.getDate() - 30);
+    const [startDate, setStartDate] = useState(initialStartDate);
+    const [endDate, setEndDate] = useState(new Date());
+    const [currentUsage, setCurrentUsage] = useState({
+        numNodes: 0,
+        numCpuUnits: 0,
+    } as SecuredUnitsUsage);
+    const [maxUsage, setMaxUsage] = useState({
+        maxNodes: 0,
+        maxNodesAt: '-',
+        maxCpuUnits: 0,
+        maxCpuUnitsAt: '-',
```

Comment:
```suggestion
        maxNodesAt: null,
        maxCpuUnits: 0,
        maxCpuUnitsAt: null,
```

Reply:
I don't think `null` is appropriate value to display in the UI, especially considering it would be rendered to empty string

Reply:
I thought I saw and even commented on the special logic for the display, which shows '-' if no time value is available.